hack/jenkins/release_update_installers.sh

```diff
@@ -19,6 +19,7 @@ git config --global user.email "minikube-bot@google.com"
 
 REPLACE_PKG_VERSION=${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_BUILD}
 REPLACE_MINIKUBE_LINUX_SHA256=$(awk '{ print $1 }' out/minikube-linux-amd64.sha256)
+REPLACE_MINIKUBE_DRIVER_KVM_SHA256=$(awk '{ print $1 }' out/docker-machine-driver-kvm2.sha256)
```

Comment:
Can you please confirm that this is the correct approach?

Reply:
Yup this should work